src/main/features/dashboard/views/macro/claimStatus/claimPartAdmitted.njk

```diff
@@ -0,0 +1,90 @@
+{% from "externalLink.njk" import externalLink %}
+
+{% macro claimPartAdmittedAlreadyPaidForDefendantDashboard(claim) %}
+  {{ t('We’ve emailed {{ defendantName }} telling them when and how you said you paid the claim.',
+    { defendantName: claim.claimData.defendant.name }) }}
+{% endmacro %}
+
+{% macro claimPartAdmittedAlreadyPaidForClaimantDashboard(claim) %}
+  {{ t('The defendant believes that they’ve paid the claim in full.') }}
```

Comment:
Is that alright? Part admission does not mean claim is paid in full. Worth checking with BA / UC.